.github/workflows/windows.yml

```diff
@@ -36,33 +36,32 @@ jobs:
         uses: actions/cache@v1.0.3
         with:
           path: C:\vcpkg\installed
-          key: ${{ matrix.os }}-vcpkg
+          key: ${{ matrix.os }}-vcpkg-v2
 
       - name: Cache Build
         uses: actions/cache@v1.0.3
         with:
           path: build
-          key: ${{ matrix.os }}-${{ matrix.EVENT_MATRIX }}-v3
+          key: ${{ matrix.os }}-${{ matrix.EVENT_MATRIX }}-v4
```

Comment:
Have you tried avoid changing cache key?

https://github.com/libevent/libevent/pull/1028#discussion_r436532618